M/mousetrap_windows/build_tarballs.jl

```diff
@@ -0,0 +1,66 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "mousetrap_windows"
+version = v"0.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Clemapfel/mousetrap.git", "ce80b9ac063170419ffd989921b7d975b74fda72"),
+    GitSource("https://github.com/Clemapfel/mousetrap_julia_binding.git", "4f389b9a4d293c71c36b055dc2316905a33fca7e"),
+    FileSource("https://dllfile.net/download/18762", "b8df8a508817b5f4d8cac6365e3fbf0e8952d50aa58ed9b5032ec5767e530aaa")
```

Comment:
As I said, from a purely technical point of view it is perfectly fine to have this `FileSource` present in all builds, and just ignore it on non-Windows build.

That doesn't change the legal and security issues, though (for the latter, at least once you are sure the DLL is legit and malware free, then the checksum should protect against the source server changing the file later on)